adnuntius/compare_json.py

```diff
@@ -1,11 +1,12 @@
+#!/usr/bin/env python3
 """Utility code to compare json objects."""
 
 __copyright__ = "Copyright (c) 2020 Adnuntius AS.  All rights reserved."
 
 try:
-    unicode
+    str
 except NameError:
-    unicode = str
+    str = str
 
```

Comment:
Pretty sure you can get rid of this, it was only required if unicode was not defined, but since we are not using unicode anymore can remove.